_scalingo_api/deployments.md

```diff
@@ -13,15 +13,16 @@ application are deployed.
 **Deployment attributes**
 
 {:.table}
-| field          | type   | description                                                     |
-| -------------- | ------ | --------------------------------------------------------------- |
-| id             | string | unique ID                                                       |
-| app_id         | string | unique ID referencing the app this deployment belongs to        |
-| created_at     | date   | date of creation                                                |
-| status         | string | status of the deployment (building, success, aborted, \*-error) |
-| git_ref        | string | git SHA                                                         |
-| pusher         | object | embedded user who pushed the GIT reference                      |
-| links          | object | hypermedia links about the deployment                           |
+| field          | type    | description                                                     |
+| -------------- | ------- | --------------------------------------------------------------- |
+| id             | string  | unique ID                                                       |
+| app_id         | string  | unique ID referencing the app this deployment belongs to        |
+| created_at     | date    | date of creation                                                |
+| status         | string  | status of the deployment (building, success, aborted, \*-error) |
+| git_ref        | string  | git SHA                                                         |
+| pusher         | object  | embedded user who pushed the GIT reference                      |
+| links          | object  | hypermedia links about the deployment                           |
+| image_size     | integer | Image size deployment (in bytes)                                |
```

Comment:
I think the following is a better phrasing, don't you think:

```suggestion
| image_size     | integer | size of the deployment image (in bytes)                                |
```

Reply:
Indeed!